index.js

```diff
@@ -49,8 +49,7 @@ class Toolbar {
         let $toolbar;
 
         let $defaultPlugins = plugins.default.map((Plugin) => { // eslint-disable-line no-unused-vars
-            // Render each plugin with the bound click handler
-            return <Plugin onClick={::this.handlePluginClick} />;
+            return <Plugin onClick={this.handlePluginClick.bind(this)} />;
```

Comment:
Oh! Good point, I hadn't considered that. Hmm...

```
onClick={(...args) => this.handlePluginClick(...args)}
```

That seems a lot worse, right?